preprocessors/ocr/ocr.py

```diff
@@ -144,7 +144,7 @@ def analyze_image(source):
         if time.time() - start_time > 3:
```

Comment:
Why do we need to call get API multiple times in a loop?
Any specific reason for this?

Reply:
since two requests are made, one to create a OCR record and one to fetch that record, we have to query the location of the record until it is created. until then it may return "in progress" in which case we have to continue waiting for its creation

Reply:
Should we even have a wait in there? Is there a downside to querying constantly in terms of price or timeouts?